### PR TITLE
[Circletracer] Disable eslint for minjs

### PR DESCRIPTION
**Pull request by CODe1995**
Add the folder containing minjs of the compressed content to eslintignore.

ONE-vscode-DCO-1.0-Signed-off-by: Seunghyun Kang <code.57x53@gmail.com>

### CODE REVIEW

**Comment by CODe1995** [comment]
from comment ([#221 comment](https://github.com/Samsung/ONE-vscode/pull/221#issuecomment-931893767))

File: .eslintignore [hunk 1]
```diff
@@ -3,3 +3,6 @@ node_modules/
 
 # Automatically generated files by the FlatBuffers compiler.
 src/Circlereader/circle-analysis/circle/
+
+# This is the folder containing the compressed minjs.
```
Comment (llFreetimell) [inline .eslintignore:7]:
```suggestion
# External open source project files which are not modified.
```

**Review by llFreetimell — APPROVED** [review]
LGTM

**Comment by llFreetimell** [comment]
@seanshpark PTAL :D

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**